srf_generation/source_parameter_generation/uncertainties/distributions.py

```diff
@@ -1,11 +1,12 @@
-from random import uniform, normalvariate
+from random import uniform
 
 import numpy as np
 from scipy.stats import truncnorm
 
 
 def relative_uniform_distribution(mean, scale_factor):
-    """A uniform distribution where the min and max are scaled relative to the middle value given as the mean"""
+    """Returns a value from a uniform distribution where the min and max
+    are scaled relative to the middle value given as the mean"""
```

Comment:
@jasonmotha Here

Reply:
Oh sweet.

Just need to fix the function names. Half of them say distribution and half of them don't.